nautobot_chatops_ipfabric/worker.py

```diff
@@ -63,14 +66,32 @@ def prompt_device_input(action_id, help_text, dispatcher, choices=None):
 
 def prompt_snapshot_id(action_id, help_text, dispatcher, choices=None):
     """Prompt the user for snapshot ID."""
-    snapshots = []
-    for snapshot in ipfabric_api.get_snapshots():
-        if snapshot["state"] == "loaded":
-            snapshot_id = snapshot["id"]
-            snapshot_name = snapshot["name"] or snapshot_id
-            snapshots.append((snapshot_name, snapshot_id))
-    choices = snapshots + [("$last", "$last")]
-    return dispatcher.prompt_from_menu(action_id, help_text, choices, default=("$last", "$last"))
+    choices = []
+    default = (LAST, LAST)
+    named_snap_ids = set()
+    snapshots = ipfabric_api.get_snapshots()
+
+    if LAST in snapshots:
```

Comment:
I'm sure we could use this code again somewhere, so it might be better served in the `ipfabric.py`. Perhaps we could create a wrapper function that calls `get_snapshots` and does some parsing to return the list of tuples containing snapshot description and ID? That would shift everything to the API module and we could set `choices` equal to the returned list.

Reply:
Moved to ipfabric.py

Reply:
Looks good. Can you run `tests` again. Seems like pydocstyle failing probably due to function docstring or something.